tests/server/controllers/userController.test.js

```diff
@@ -21,16 +21,28 @@ const addUsersUrl = '/api/v1/users/addusers';
 const getUsersUrlStudent = '/api/v1/users/student';
 const getUsersUrlTeacher = '/api/v1/users/teacher';
 const getUsersUrlInvalid = '/api/v1/users/stu';
+const updateUserUrl = '/api/v1/users/update';
 
 let userSession = '';
 
-
 describe("User Controller", () => {
     before(async () => {
         try {
-            await Users.create({
+            await Users.bulkCreate([{
+                user_uid: '40e6215d-b5c6-4896-987c-f30f3678f607',
+                school_uid: '40e6215d-b5c6-4896-987c-f30f3678f609',
```

Comment:
@jamsgra Can we have all these user objects in the `mockData` directory? The file `userController.js` is getting too long to read.

Reply:
Sounds good